### DISCUSSION TITLE
[HELP] Train customized dataset with glowtts

### DISCUSSION THREAD

**Discussion by cdlIris** [coqui-ai/TTS#2164] [General Q&A]
I'm trying to train the glow-tts using my customized voice with ljspecch text. There are a few questions that I'm not sure:

- What is the final loss?
       - When I checked the repo from jaywalnut310, the final loss was -0.08797, and the steps were 240k. 
       - But when I checked the pretrained model provided by coqui-ai, it was 330k, and  unknown loss
- Is the  pre-trained vocoder provided still compatible with the customized dataset? the sampling rate is the same as the LJSpeech
       - Is the glow-tts compatible with the hifigan? Is there a notebook that can be used as reference? 
- My current config of training has `add_blank` set to true. Can anybody provide a pre-trained model that set `add_blank` to test? 

Thank you for your help!
